### PR TITLE
Language and linter checks testing

### PR DESCRIPTION
**Pull request by Zabuzard**
just a test

### CODE REVIEW

**Comment by marko-radosavljevic** [comment]
Fixed languagetool not finding typos. ^^

It's unclear how well it will behave on real world use-cases, this setup is very experimental so I will play with it more.

Some of the potential issues I noticed:
- We have limitations on the api and checking power: https://dev.languagetool.org/public-http-api
    - And it doesn't help that this langaugetool action scans every single file, instead only files that have been modified.
- Action is 'In progress' for 20mins already, unsure if it's broken.
- Not sure how well it will work on the code and markdown, although all checks that would trigger on code syntax are disabled, like underscores and similar.

**Comment by marko-radosavljevic** [comment]
Action is erroring out with 'Only 65535 characters are allowed; 97549 were supplied. []', because the request is too large for github's liking.

I believe changing _reporter_ to `github-pr-review` might solve this problem.

I think review comments are much better anyway, because we can discuss if it's a real mistake or something that should be ignored. If it could suggest fixes that you can merge from the UI, it would be a really nice experience.
Also, with review comments, you can 'resolve' them to hide them from the review UI, so it's not cluttering everything.

Currently, it looks like this:
image

File: src/integers.md [hunk 1]
```diff
@@ -8,4 +8,12 @@
 int z = -4;
 ```
 
+### No
 
+sdsd
+* dasd
+
+* ddd
```
[the remark this answers is not in the document]
Comment (marko-radosavljevic) [reply src/integers.md:16]: jesus, it doesn't work nicely with random characters. :smile: 

and I don't think there is any easy way to limit the amount of suggestions api returns

**Comment by marko-radosavljevic** [comment]
It didn't crash, seems to be working well now. :relaxed:

Definitely like review reporter much more than the check reporter, will change linter to match it at some point as well.

**Comment by Zabuzard** [comment]
the reason i didnt go with that setting it is bc people can bypass it by just resolving the convo. so its impossible to make it "required".

but i think it would be fine anyways

**Comment by bowbahdoe** [comment]
Closing this. Linter changes will happen in a new PR

**Pull request closed without merging**